pending/contributing.pod

```diff
@@ -0,0 +1,122 @@
+(that's not to say your report will get ignored there!).
+
+Development of Dancer happens on Github. Your best bet for reporting a bug
+in Dancer is to go to the L<issue tracker on GitHub|https://github.com/PerlDancer/Dancer2/issues>.
+Before submitting a new bug, please be courteous and look to see if there
+is already an open issue for the problem you are observing, and if so, add
+whatever additional information you have to the existing issue.
+
+No matter how well we test Dancer during development, there is no way for us
+to find all of the potential bugs that may arise. The Core Team and the Dancer
+community depends on B<you> to report bugs that you find along the way!
+
+=head2 IRC
+
+A great way to participate and contribute to the Dancer project is to be active
+on the IRC channel. The Dancer Core Team strives to maintain a friendly and civil
+environment on the channel at all times, and users of C<#dancer> would generally
+agree that it's a pretty friendly place to be.
+
+To get started, all you have to do is point your favorite IRC client to
+C<irg.perl.org#dancer> and you're off and running! Ask a question, answer
+a question, pitch your plugin idea... there are a number of ways for you 
+to get involved.
+
+All users are protected by our 
+L<community policy and community standards|https://metacpan.org/pod/distribution/Dancer2/lib/Dancer2/Policy.pod>
+document. The Core Team takes violations of this policy very, very seriously.
+
+=head2 Sending a Documentation Patch
+
+This is one of the easiest ways to get involved, and how yours truly got his
+start in the Dancer community (so I know for a fact this is an effective 
+way!). We as developers are not always the best at documenting our own work,
+no matter how hard we try. We are biased, and used to looking at our work 
+in a particular way, and we cannot always explain how something should be 
+used in the best way, or document things in such a way that it answers all
+the questons a user may have. Dancer2 is no exception.
+
+Did you find a particular area of the docs to be hard to understand? Clarify
+it and send us a pull request on GitHub. Did we forget to document a 
+method or configuration setting? Send us a patch! Dancer's documentation is 
+a living work, and will evolve and improve over time with the help of our
+community. 
+
+Another useful addition to the documentation is to add a recipe to the 
+L<cookbook|Dancer2::Cookbook>. It's a great place to show a useful tip that
+doesn't really fit elsewhere in the documentation.
+
+=head2 Taking an Issue from the GitHub Queue
+
+At the time of this writing, there are 82 current issues in the GitHub queue,
+13 of which are labelled Beginner Suitable. A great way to help out the 
+Dancer Core Team and the community is to pick up one of these issues, hack it
+out, and send us a pull request. No matter what your level of expertise is,
+there is something in the issue queue that will be suitable for you.
```

Comment:
I would be happy if there was a comment on helping out people  who are not familiar with Github. We accept patches in any format, but we will also be happy to help people with their first PR.